src/server/cluster/cluster_slot_migration.cc

```diff
@@ -142,8 +142,7 @@ void ClusterSlotMigration::MainMigrationFb() {
                  << ec.message();
   }
 
-  bool success = true;
-  if (success = IsFinalized(); success) {
+  if (IsFinalized()) {
     state_ = MigrationState::C_FINISHED;
```

Comment:
update state after you get ok?

Reply:
I think it's the correct place to set a new status, ACK cmd and OK doesn't affect slot migration process, and more like error processing